source/lib/rocprof-sys/library/rocprofiler-sdk.cpp

```diff
@@ -827,6 +827,121 @@ tool_tracing_buffered(rocprofiler_context_id_t /*context*/,
                                           _end_ns);
                 }
             }
+            else if(header->kind == ROCPROFILER_BUFFER_TRACING_HIP_RUNTIME_API)
+            {
+                auto* record = static_cast<rocprofiler_buffer_tracing_hip_api_record_t*>(
+                    header->payload);
+
+                if(record->operation == ROCPROFILER_HIP_RUNTIME_API_ID_hipEventRecord)
```

Comment:
```suggestion
                const auto _ops = std::unordered_set<int32_t>{
                    ROCPROFILER_HIP_RUNTIME_API_ID_hipEventRecord,
                    ROCPROFILER_HIP_RUNTIME_API_ID_hipEventSynchronize,
                };
                
                if(_ops.count(record->operation) > 0)
```

It looks like both handlers are identical. Can you do something like this and then remove the "else-if" block?
Are there other operations that we want to handle?